### PR TITLE
Kempe's Algorithm in Python implemented

### PR DESCRIPTION
**Pull request by Riyazul555**
Closes PR  #47

### CODE REVIEW

**Comment by Riyazul555** [comment]
I have resolved all the required changes....please see it once @iuliagroza before I merge it.....Thanks

File: Kempe/Kempe.py [hunk 1]
```diff
@@ -2,15 +2,15 @@
 # This code is contributed by Riyazul555
 
 
-# Time Complexity = O(N^3) (approximate)
+# Time Complexity = O(N * M * f(N, M))
```
Comment (iuliagroza) [inline Kempe/Kempe.py:5]: The time complexity should be the same no matter the language used for implementation, so maybe O(2^N)?

File: Kempe/Kempe.py [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 # Space Complexity = O(N)
```
Comment (iuliagroza) [inline Kempe/Kempe.py:6]: Please fix the space complexity as well.

File: Kempe/Kempe.py [hunk 3]
```diff
@@ -23,37 +23,26 @@ def bfs(nod, G, viz):
...
-        p = k % MOD
-        for i in range(1, n):
-            p = (p % MOD * (k - 1) % MOD) % MOD
-        return p
-
-    G1 = [[] for _ in range(nMax)]
-    ind = 0
-    while not G[vert[ind]]:
-        ind += 1
+        return pow(k, n, MOD)
```
Comment (iuliagroza) [inline Kempe/Kempe.py:36]: Nice one!

File: Kempe/Kempe.py [hunk 4]
```diff
@@ -76,7 +65,7 @@ def chromatic_polynomial(k, n, m, G, vert):
                 if j not in G2[i]:
                     G2[i].append(j)
     sz = sum(len(G2[i]) for i in vert2) // 2
-    return (chromatic_polynomial(k, n, m - 1, G1, vert) % MOD - chromatic_polynomial(k, n - 1, sz, G2, vert2) % MOD + MOD) % MOD
+    return (chromatic_polynomial(k, n, m - 1, G1, vert) - chromatic_polynomial(k, n - 1, sz, G2, vert2) + MOD) % MOD
```
Comment (iuliagroza) [inline Kempe/Kempe.py:68]: I think the modular subtraction formula is (A-B)%MOD = (A%MOD - B%MOD)%MOD, so no need to remove those `MOD`s, but we can remove the added one.

**Review by iuliagroza — APPROVED** [review]
Nice changes! LGTM! You can close the issue after fixing these minor changes.

**Comment by Riyazul555** [comment]
I have fixed the finishing changes also .....merging this pR @iuliagroza

**Pull request merged**